src/migration/service/knexMigrator.ts

```diff
@@ -90,6 +92,20 @@ export async function resolveMigrationContext(
 
       return new SqlMigrationSourceContext(src);
 
+    case 'javascript':
+      const srcJS = await resolveJavaScriptMigrations(migrationPath);
+
+      log('Available migration sources:\n%O', srcJS);
+
+      return new JavaScriptMigrationContext(srcJS);
+
+    case 'typescript':
+      const srcTS = await resolveJavaScriptMigrations(migrationPath, FileExtensions.TS);
+
+      log('Available migration sources:\n%O', srcTS);
+
+      return new JavaScriptMigrationContext(srcTS);
```

Comment:
It's cleaner this way. 
```suggestion
    case 'javascript':
      const src = await resolveJavaScriptMigrations(migrationPath);

      log('Available migration sources:\n%O', src);

      return new JavaScriptMigrationContext(src);

    case 'typescript':
      const src = await resolveJavaScriptMigrations(migrationPath, FileExtensions.TS);

      log('Available migration sources:\n%O', src);

      return new JavaScriptMigrationContext(src);
```